Sources/SwiftSyntax/Trivia.swift

```diff
@@ -42,6 +42,28 @@ public struct Trivia: Sendable {
     pieces.isEmpty
   }
 
+  /// The string contents of all the comment pieces with any comments tokens trimmed.
+  public var commentValue: String {
+    var comments = [String]()
+
+    for piece in pieces {
+      switch piece {
+      case .blockComment(let text), .docBlockComment(let text):
+        let text = text.trimmingCharacters(in: "/* ").trimmingAnyCharacters(in: ["\n"])
```

Comment:
Should we only trim `/*` as a prefix? If you have a comment with `/* ///// abc */`, I would expect the `commentValue` to be `///// abc`, not `abc`